test/test_sort_and_select.py

```diff
@@ -113,6 +113,77 @@ def test_sort(self, device):
         self.assertIsOrdered('descending', x, res2val, res2ind,
                              'random with NaNs')
 
+    @onlyCPU
+    @dtypes(*set(torch.testing.get_all_dtypes()) - {torch.bfloat16, torch.complex64, torch.complex128})
+    def test_stable_sort(self, device, dtype):
```

Comment:
Add a test that the expected error message is thrown on CUDA when stable=True is passed, but not when stable=False is passed.